src/type-layout.md

```diff
@@ -304,6 +304,11 @@ the default `enum` size and alignment for the target platform's C ABI.
 > Note: The enum representation in C is implementation defined, so this is
 > really a "best guess". In particular, this may be incorrect when the C code
 > of interest is compiled with certain flags.
+>
+> However, it is guaranteed that, for any two `repr(C)` enum types, their
+> discriminants will have the same size and alignment *within a given
+> compilation of the program.* This is true for both field-less enums and
+> for enums with fields.
```

Comment:
You probably mean the tag, not the discriminant. The discriminant is the value you get when you cast an enum to an integer or call `std::mem::discriminant` on it. The tag is the actual representation in memory. It may be smaller or larger than the discriminant. Without `repr(C)` or `repr(iN)` the tag may be a different integer than the discriminant or the discriminant may be represented using niches instead.